app/src/main/java/com/ccc/nameapp/data/source/local/sharedprf/SharedPrefsApi.kt

```diff
@@ -21,6 +21,7 @@ class SharedPrefsImpl @Inject constructor(context: Context) : SharedPrefsApi {
         Context.MODE_PRIVATE
     )
 
+    @Suppress("UNCHECKED_CAST")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/9cbff12fc6911fd41007439107765dbec2127af9/app/src/main/java/com/ccc/nameapp/data/source/local/sharedprf/SharedPrefsApi.kt#L24"></span>The function get appears to be too complex.</td>
    </tr>
  </tbody>
</table>